bindings/pyroot/pythonizations/python/ROOT/pythonization/_roofit/_roomsgservice.py

```diff
@@ -11,41 +11,29 @@
 ################################################################################
 
 
-r"""
-/**
-\class RooMsgService
-\brief \parblock \endparblock
-\htmlonly
-<div class="pyrootbox">
-\endhtmlonly
-
-## PyROOT
-
-Some member functions of RooMsgService that take a RooCmdArg as argument also support keyword arguments.
-So far, this applies to RooMsgService::addStream.
-For example, the following code is equivalent in PyROOT:
-\code{.py}
-# Directly passing a RooCmdArg:
-ROOT.RooMsgService.instance().addStream(
-    ROOT.RooFit.DEBUG, ROOT.RooFit.Topic(ROOT.RooFit.Tracing), ROOT.RooFit.ClassName("RooGaussian")
-)
-
-# With keyword arguments:
-ROOT.RooMsgService.instance().addStream(ROOT.RooFit.DEBUG, Topic = ROOT.RooFit.Tracing, ClassName = "RooGaussian")
-\endcode
-
-\htmlonly
-</div>
-\endhtmlonly
-*/
-"""
-
-from ._utils import _kwargs_to_roocmdargs
+from ._utils import _kwargs_to_roocmdargs, cpp_signature
 
 
 class RooMsgService(object):
+    """Some member functions of RooMsgService that take a RooCmdArg as argument also support keyword arguments.
+    So far, this applies to RooMsgService::addStream.
+    For example, the following code is equivalent in PyROOT:
+    \code{.py}
+    # Directly passing a RooCmdArg:
+    ROOT.RooMsgService.instance().addStream(ROOT.RooFit.DEBUG, ROOT.RooFit.Topic(ROOT.RooFit.Tracing), ROOT.RooFit.ClassName("RooGaussian"))
+
+    # With keyword arguments:
+    ROOT.RooMsgService.instance().addStream(ROOT.RooFit.DEBUG, Topic = ROOT.RooFit.Tracing, ClassName = "RooGaussian")
+    \endcode"""
+
+    @cpp_signature(
+        "Int_t addStream(RooFit::MsgLevel level, const RooCmdArg& arg1=RooCmdArg(), const RooCmdArg& arg2=RooCmdArg(), const RooCmdArg& arg3=RooCmdArg(),"
```

Comment:
```suggestion
        "Int_t RooMsgService::addStream(RooFit::MsgLevel level, const RooCmdArg& arg1=RooCmdArg(), const RooCmdArg& arg2=RooCmdArg(), const RooCmdArg& arg3=RooCmdArg(),"
```
Class name is missing.